mitodl/hugo-course-publisher
# add favicon

#### Pre-Flight checklist


#### What are the relevant tickets?

none

#### What's this PR do?

just grabbed the favicon from prod OCW and added it so we don't have to look at default one :)

#### How should this be manually tested?

just check out the branch, do a no-cache refresh, and make sure you see the favicon!
